### PR TITLE
BIG-3649 More specificity for failures

### PR DESCRIPTION
**Pull request by lindseyreno**
This is part one of this ticket.  Once it is merged I will update aardwolf and verify the counters update there.
@dlecocq @b4hand

### CODE REVIEW

File: reppy/exceptions.py [hunk 1]
```diff
@@ -36,3 +36,22 @@ def __init__(self, *args, **kwargs):
             self.status = args[1]
         ReppyException.__init__(self, *args, **kwargs)
 
+class SSLException(ReppyException):
+    def __init__(self, *args, **kwargs):
```
Comment (dlecocq) [inline reppy/exceptions.py]: I imagine this could be written just as:

```python
class SSLException(ReppyException):
    pass
```

Comment (lindseyreno) [inline reppy/exceptions.py]: Ok, just this one because the message is really clear or all of the new exceptions?

Comment (dlecocq) [inline reppy/exceptions.py]: For all of them -- their `__init__` simply invokes the parent, so `pass` should do the same thing less verbosely.

Comment (lindseyreno) [inline reppy/exceptions.py]: Ah ok, done.

**Pull request merged**